addons/project/static/tests/tours/project_update_tour_tests.js

```diff
@@ -0,0 +1,141 @@
+odoo.define('project.update.tour', function (require) {
```

Comment:
Why don't you create the module with the new module definition ? 🙂

Reply:
Done :o

Reply:
Thanks ! 😉